packages/matter.js/src/cluster/ActionsCluster.ts

```diff
@@ -0,0 +1,345 @@
+    actionId: TlvField(0, TlvUInt16),
+    invokeId: TlvOptionalField(1, TlvUInt32),
+});
+
+/** @see {@link MatterCoreClusterSpecificationV1_0} § 9.14.7.7 */
+const TlvPauseActionWithDurationRequest = TlvObject({
+    actionId: TlvField(0, TlvUInt16),
+    invokeId: TlvOptionalField(1, TlvUInt32),
+    duration: TlvField(2, TlvUInt32),
+});
+
+/** @see {@link MatterCoreClusterSpecificationV1_0} § 9.14.7.8 */
+const TlvResumeActionRequest = TlvObject({
+    actionId: TlvField(0, TlvUInt16),
+    invokeId: TlvOptionalField(1, TlvUInt32),
+});
+
+/** @see {@link MatterCoreClusterSpecificationV1_0} § 9.14.7.9 */
+const TlvEnableActionRequest = TlvObject({
+    actionId: TlvField(0, TlvUInt16),
+    invokeId: TlvOptionalField(1, TlvUInt32),
+});
+
+/** @see {@link MatterCoreClusterSpecificationV1_0} § 9.14.7.10 */
+const TlvEnableActionWithDurationRequest = TlvObject({
+    actionId: TlvField(0, TlvUInt16),
+    invokeId: TlvOptionalField(1, TlvUInt32),
+    duration: TlvField(2, TlvUInt32),
+});
+
+/** @see {@link MatterCoreClusterSpecificationV1_0} § 9.14.7.11 */
+const TlvDisableActionRequest = TlvObject({
+    actionId: TlvField(0, TlvUInt16),
+    invokeId: TlvOptionalField(1, TlvUInt32),
+});
+
+/** @see {@link MatterCoreClusterSpecificationV1_0} § 9.14.7.12 */
+const TlvDisableActionWithDurationRequest = TlvObject({
+    actionId: TlvField(0, TlvUInt16),
+    invokeId: TlvOptionalField(1, TlvUInt32),
+    duration: TlvField(2, TlvUInt32),
+});
+
+/**
+ * Provides feedback to the client about the progress of the action.
+ * @see {@link MatterCoreSpecificationV1_0} § 9.14.8.1
+ */
+const StateChangedEvent = {
+    actionId: TlvField(0, TlvUInt16),
+    invokeId: TlvField(1, TlvUInt32),
+    newState: TlvField(2, TlvEnum<ActionStateEnum>()),
+};
+
+/**
+ * Provides feedback to the client about the non-successful progress of the action.
+ * @see {@link MatterCoreSpecificationV1_0} § 9.14.8.2
+ */
+const ActionFailedEvent = {
+    actionId: TlvField(0, TlvUInt16),
+    invokeId: TlvField(1, TlvUInt32),
+    newState: TlvField(2, TlvEnum<ActionStateEnum>()),
+    error: TlvField(3, TlvEnum<ActionErrorEnum>()),
+};
+
+/**
+ * This cluster provides a standardized way for a Node (typically a Bridge, but could be any Node) to expose
+ *  * information about logical grouping of endpoints on the Node (example: lights in a room)
+ *  * information about named actions that can be performed on such a group of endpoints (example: recall a
+ *    scene for a group of lights by its name)
+ *  * commands to trigger such actions
+ *  * events to receive feedback on the state of such actions.
+ *
+ * @see {@link MatterCoreSpecificationV1_0} § 9.14
+ */
+export const ActionsCluster = Cluster({
+    id: 0x25,
+    name: "Actions",
+    revision: 1,
+
+    /** @see {@link MatterCoreSpecificationV1_0} § 9.14.6 */
+    attributes: {
+        /** List of actions. */
+        actionList: Attribute(0, TlvArray(TlvAction), { default: [] }),
```

Comment:
please add {maxLength: 256} in the TlvArray as second parameter

Reply:
Done.